pipelines/LSSTCam/SingleFrame.yaml

```
@@ -0,0 +1,6 @@
+description: >-
+  Single-frame pipeline for the case in which
+  no templates exist, specialized for LSSTCam
```

Again, no need to assume _why_ the user might want to run it (it's not like having templates will somehow invalidate the pipeline!):
```suggestion
  Standalone single-frame pipeline, specialized for LSSTCam.
```